docs/specs/lcb-packages.md

```diff
@@ -0,0 +1,246 @@
+# LiveCode Builder Packages
+Copyright 2015 LiveCode Ltd.
+
+## Introduction
+
+LiveCode Builder is built around the idea of 'modules' - independently compiled
+units of code which can be selectively used by others.
+
+In a larger system, however, just the compiled module code is not enough. There
+are other things which are naturally related to the module which don't make sense
+to be included within the modules binary file - principally because they are not
+of direct interest to the virtual machine which must execute the code.
+
+To solve this problem, compiled LCB modules are embedded in a more general
+container called a 'package'. A package is a ZIP format archive file with a
+standard structure which combines modules together with other related resources
+and metadata which can be used by an environment loading it.
+
+## Structure
+
+An LCB package is a ZIP archive with a standard layout:
+
+    <root>/

```

Comment:
<root> is the root of the zip archive - so a package org.example would be named org.example.zip with the manifest file at top-level manifest.xml inside org.example.zip.